lib/l10n.js

```diff
@@ -13,456 +13,569 @@ var l10n = {
+        es: "Lo siento su inscripción no se pudo procesar. Diga PROCESA DE NUEVO para procesarlo otra vez, o EMPEZAR DE NUEVO para empezar desde el principio",
+        ko: "등록이 완료되지 않았습니다. 다시 시도하기 위해서는 RETRY 라고 입력하세요. 처음부터 다시 입력하기 위해서는 RESTART 라고 입력하세요."
     },
     msg_share: {
         en: "Help your friends get registered -- please share HelloVote!\n* Share on Facebook: hellovote.org/share\n* Share on Twitter: {tweet_url}",
-        es: "Porfavor comparta HolaVote!\n* Comparta en Facebook: https://hellovote.org/share\n* Comparta en Twitter: {tweet_url}"
+        es: "Porfavor comparta HolaVote!\n* Comparta en Facebook: https://hellovote.org/share\n* Comparta en Twitter: {tweet_url}",
+        ko: "HelloVote에 대해 주변에 홍보 해 주세요!\n* 페이스북 공유: hellovote.org/share \n* 트위터 공유: {tweet_url}"
     },
     msg_share_sms: {
         en: "Forward this message to everyone you care about! Register to vote by text message: Visit https://hello.vote",
-        es: "¡Reenvíe el mensaje siguiente a sus amigos! Registrese para votar por mensaje de texto: Visite https://hello.vote"
+        es: "¡Reenvíe el mensaje siguiente a sus amigos! Registrese para votar por mensaje de texto: Visite https://hello.vote",
+        ko: "간편한 스마트폰 유권자 등록에 대해 널리 알려주세요! http://hello.vote 를 방문하시면 시작 할 수 있습니다."
     },
     msg_sms_notice: {
         en: "Fight for the Future (FFTF) & its Education Fund (FFTFEF) will text you voting information and other action alerts. To stop messages, text STOP.",
-        es: "Fight for the Future (FFTF) y su Fondo de Educación (FFTFEF) le enviará información de votante y otras alertas de acción. Para anular su suscripción envíe STOP. "
+        es: "Fight for the Future (FFTF) y su Fondo de Educación (FFTFEF) le enviará información de votante y otras alertas de acción. Para anular su suscripción envíe STOP. ",
+        ko: "이 앱의 주관 단체인 FFTF 및 FFTFEF (\“미래를 쟁취하자]\”) 에서 선거 정보를 포함한 여러가지 정보를 보내드릴 것입니다. 메세지를 그만 받기를 원하시면 STOP 을 입력하세요."
     },
     msg_sms_notice_partner: {
         en: "Fight for the Future (FFTF), its Education Fund (FFTFEF) & {{partner}} will text you voting information and other action alerts.",
-        es: "Fight for the Future (FFTF), su Fondo de Educación (FFTFEF), y {{partner}} le enviará información de votante y otras alertas de acción."
+        es: "Fight for the Future (FFTF), su Fondo de Educación (FFTFEF), y {{partner}} le enviará información de votante y otras alertas de acción.",
+        ko: "이 앱의 주관 단체인 FFTF 및 FFTFEF (\“미래를 쟁취하자]\”) 에서 선거 정보를 포함한 여러가지 정보를 보내드릴 것입니다. "
     },
     msg_sms_fftf_stop: {
         en: "(To stop messages from FFTF & FFTFEF, text STOP.)",
-        es: "(Para detener mensajes de FFTF & FFTFEF, envíe STOP.)"
+        es: "(Para detener mensajes de FFTF & FFTFEF, envíe STOP.)",
+        ko: " 이 앱의 주관 단체인 FFTF 및 FFTFEF (\“미래를 쟁취하자]\”) 에서 선거 정보를 포함한 여러가지 정보를 보내드릴 것입니다. "
```

Comment:
The opt-out keyword needs to be STOP, not the translation into Korean. This is a legal requirement.

Reply:
asking for a rewrite that uses STOP in English